### PR TITLE
Fixed `--volumes` validation

### PR DESCRIPTION
**Pull request by surajnarwade**

### CODE REVIEW

File: pkg/app/app.go [hunk 1]
```diff
@@ -76,6 +76,9 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	replicationController := cmd.Flags().Lookup("replication-controller").Changed
 	deployment := cmd.Flags().Lookup("deployment").Changed
 
+	//Check volume flag
+	volume := cmd.Flags().Lookup("volumes").Value.String()
```
Comment (cdrage) [inline pkg/app/app.go]: No need to do a look up. It's available at `opt.Volume`

File: pkg/app/app.go [hunk 2]
```diff
@@ -133,6 +136,10 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	if opt.GenerateJSON && opt.GenerateYaml {
 		log.Fatalf("YAML and JSON format cannot be provided at the same time")
 	}
+
+	if volume != "persistentVolumeClaim" && volume != "emptyDir" {
```
Comment (cdrage) [inline pkg/app/app.go]: change volume to `opt.Volume`

**Comment by surajnarwade** [comment]
@cdrage , @kadel needs your review here :)

**Comment by kadel** [comment]
ref https://github.com/kubernetes/kompose/issues/814

File: pkg/app/app.go [hunk 3]
```diff
@@ -133,6 +133,10 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	if opt.GenerateJSON && opt.GenerateYaml {
 		log.Fatalf("YAML and JSON format cannot be provided at the same time")
 	}
+
+	if opt.Volumes != "persistentVolumeClaim" && opt.Volumes != "emptyDir" {
+		log.Fatal("Unknown Volume type: ", opt.Volumes)
```
Comment (kadel) [inline pkg/app/app.go]: Just one small note here. It would be nice if message also showed what values are possible.

File: pkg/app/app.go [hunk 4]
```diff
@@ -133,6 +133,10 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	if opt.GenerateJSON && opt.GenerateYaml {
 		log.Fatalf("YAML and JSON format cannot be provided at the same time")
 	}
+
+	if opt.Volumes != "persistentVolumeClaim" && opt.Volumes != "emptyDir" {
+		log.Fatal("Unknown Volume type: ", opt.Volumes, ", Possible values are: persistentVolumeClaim, emptyDir")
```
Comment (cdrage) [inline pkg/app/app.go]: remove capitalization of `Possible`

change to `persistentVolumeClaim and emptyDir`

**Comment by surajnarwade** [comment]
@cdrage done

**Comment by cdrage** [comment]
@surajnarwade Please add description to commit.

**Comment by surajnarwade** [comment]
issue is being tracked here https://github.com/kubernetes/kompose/issues/814

**Comment by cdrage** [comment]
@surajnarwade can you add a test? unit / cli test (make sure it fails if someone puts --volumes=foobar or something)

**Pull request merged**